index.js

```diff
@@ -63,6 +64,21 @@ app.post('/submit', async (req, res) => {
     res.status(200).send()
 })
 
+/**
+ * Endpoint for validation of orders. Used by the OME to validate signatures before 
+ * passing on the orders to the book. This rejects invalid orders early to avoid
+ * false liquidity on the books.
+ */
+app.post('/check', async (req, res) => {
+    if (!req.body.order || !req.body.trader || !req.body.network || !req.body.sig) {
+        console.log("missing params")
+        return res.status(500).send({ error: "Invalid params provided" })
```

Comment:
No server error has occurred, this should be a [400](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400).